### ISSUE TITLE
Should Liberapay strive towards a flat payment processing fee?

### ISSUE THREAD

**Issue by Changaco** [liberapay/salon#356]
If it was possible for Liberapay to have a flat 5% fee for all payments through Stripe, should we adopt that system instead of continuing to encourage donors to send more money in advance to lower the fee percentage?

A flat fee basically means that wealthier people pay higher fees so that poorer folks pay lower fees. Of course the downside of this kind of system is that it's less attractive to big donors, which is potentially problematic.

**Comment by trebmuh** [comment]
What is the percentage average with the current system?

**Comment by Changaco** [comment]
Less than 3%: <https://github.com/liberapay/liberapay.com/issues/346#issuecomment-476110074>.

**Comment by MartinDelille** [comment]
I agree with the fixed flat fee. It would make the communication simpler for the donor and worth paying a little bit extra. If we have an explicit communication about what the 5% repartition we should be fine with critics.

**Comment by trebmuh** [comment]
There is something I don't get, sorry if I'm becoming dumb here, but the idea sounds as if we would go from a 3% average fee to a 5% one?

**Comment by Changaco** [comment]
I said 5% because it's Stripe's standard fee in Japan, but they might agree to a lower percentage since our current average is below 3%.

**Comment by trebmuh** [comment]
OK, thanks for the precisions. I think it'd be good as long as it doesn't increase the average fee we are already paying to them.

**Comment by Changaco** [comment]
It just occurred to me that a significant point hasn't been mentioned in this discussion: a fixed-percentage fee would partially solve the roadblock described in <https://github.com/liberapay/liberapay.com/issues/839#issuecomment-454007104>.

**Comment by mattbk** [comment]
I agree that a flat fee would simplify communication. Glad the average was already added to the FAQ. It should just be emphasized that even with a flat fee, Liberapay isn't making a profit.
